### PR TITLE
Add `inspector` implementation

### PR DESCRIPTION
**Pull request by stefanDeveloper**
Current implementation of `inspector`

### CODE REVIEW

File: requirements-dev.txt
Comment (lamr02n) [inline requirements-dev.txt:1]: Can we safely delete the requirements ``confluent_kafka`` and ``colorlog`` here? We still need them for the other modules. This depends on the fact if the ``requirements-dev.txt`` is still used for those modules (or if the respective ``requirements.*.txt`` is sufficient). Have you tested it without these entries?

**Review by lamr02n — CHANGES_REQUESTED** [review]
Please check the comment on the ``requirements-dev.txt``. Additionally, there seems to be a problem with the tests. One of them is in the ``inspector.get_and_return_data``:
```
======================================================================
ERROR: test_get_data_with_return_data (test_inspector.TestGetData)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/unittest/mock.py", line 1379, in patched
    return func(*newargs, **newkeywargs)
  File "/home/runner/work/heiDGAF/heiDGAF/tests/test_inspector.py", line 63, in test_get_data_with_return_data
    sut.get_and_fill_data()
  File "/home/runner/work/heiDGAF/heiDGAF/src/inspector/inspector.py", line 101, in get_and_fill_data
    self.messages = [ast.literal_eval(item) for item in data.get("data")]
  File "/home/runner/work/heiDGAF/heiDGAF/src/inspector/inspector.py", line 101, in <listcomp>
    self.messages = [ast.literal_eval(item) for item in data.get("data")]
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/ast.py", line 110, in literal_eval
    return _convert(node_or_string)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/ast.py", line 109, in _convert
    return _convert_signed_num(node)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/ast.py", line 83, in _convert_signed_num
    return _convert_num(node)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/ast.py", line 74, in _convert_num
    _raise_malformed_node(node)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/ast.py", line 71, in _raise_malformed_node
    raise ValueError(msg + f': {node!r}')
ValueError: malformed node or string on line 1: <ast.Name object at 0x7ff1812c94e0>
```
For Python 3.9, it looks like there is a problem with importing libraries.

File: requirements-dev.txt
Comment (stefanDeveloper) [reply requirements-dev.txt:1]: Works fine!

**Review by lamr02n — APPROVED** [review]
LGTM

**Pull request merged**